### PR TITLE
[OSDEV-1171] Decrease RDS instance type for staging and test

### PR DESCRIPTION
**Pull request by killev**
- RDS instances for `staging` and `test` have beed decreased to `db.t3.large`

### CODE REVIEW

File: deployment/environments/terraform-test.tfvars [hunk 1]
```diff
@@ -16,7 +16,7 @@ bastion_instance_type = "t3.nano"
 rds_allocated_storage = "128"
 rds_engine_version = "12"
 rds_parameter_group_family = "postgres12"
-rds_instance_type = "db.t3.2xlarge"
+rds_instance_type = "db.t3.2large"
```
Comment (VadimKovalenkoSNF) [inline deployment/environments/terraform-test.tfvars:19]: Should it be `db.t3.large` instead?

**Review by VadimKovalenkoSNF — CHANGES_REQUESTED** [review]
Looks like a typo here:

**Review by VadimKovalenkoSNF — APPROVED** [review]
LGTM

**Review by vladsha-dev — APPROVED** [review]
LGTM

**Pull request merged**